apps/imageio/src/main/java/imageio/Main.java

```diff
@@ -119,7 +119,7 @@ public static void paintRectangles() throws IOException, FontFormatException {
 
     private static void resizeImage() throws IOException {
         final BufferedImage img = ImageIO.read(Main.class.getResourceAsStream("/Grace_M._Hopper.jp2"));
-        int height = 50;
+        int height = 555;
```

Comment:
This change seems irrelevant to the rest. Is it on purpose?

Reply:
Yes, it's on purpose to normalize the size o the tested pictures so as we don't have to pick a different set of pixels (out of bounds).